advanced_html_css/responsive_design/introduction_to_responsive_design.md

```diff
@@ -20,12 +20,12 @@ There are ways to make your designs flexible enough that they work well on most
 The smallest phones in common circulation rarely get smaller than `320px`, so that should serve as a reliable lower end target. If your website works as low as `320px` width, then it should work on any small device. The upper limit is a little harder to clarify. These days super-duper-ultra-wide monitors aren't uncommon, so you should plan for the possibility that your site could be viewed on some ridiculously wide monitor. Often, this is accomplished by setting up a max-width for _all_ of your content and then centering that on the page. With a setup like that your content will look fine even on super-wide resolutions.
 
 
```

Comment:
nitpick: Looks like this file had an extra empty line at line 22. Shouldn't impact how the lesson displays on the site but would you mind removing that one while we're at it and then I can push this through?

Thanks ab! :-)

Reply:
done, please have a look, thanks :)